.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "liveServer.settings.port": 5501
```

Comment:
You can delete this while creating a PR

Reply:
ok

Reply:
also made the search bar width change in component-library, it will shortly reflect here after merging that one